lib/react_on_rails/configuration.rb

```diff
@@ -156,7 +161,6 @@ def initialize(node_modules_location: nil, server_bundle_js_file: nil, prerender
       self.webpack_generated_files = webpack_generated_files
       self.rendering_extension = rendering_extension
 
-      self.server_render_method = server_render_method
```

Comment:
Add a message, if the value is provided that it's been removed and to contact ShakaCode.com for React on Rails Pro.